resources/apt-packages.txt

```diff
@@ -0,0 +1,38 @@
+# utilities
+git
+mercurial
+pkg-config
+wget
+# debian-provided interpreters
+python-pip
+python2.7
+python2.7-dev
```

Comment:
A lot of the packages in this list are needed for building python 3.5 but are no longer needed at runtime.  It wouldn't be a horrible idea to remove them once you're done.  In my experience, if you provide something in your runtime, some enterprising user will find a use for it and come to depend on it.  Then, removing it later becomes hard.